Makefile

```diff
@@ -1571,7 +1571,7 @@ ovn_kuttl_run: ## runs kuttl tests for the ovn operator, assumes that everything
 ovn_kuttl: export NAMESPACE = ${OVN_KUTTL_NAMESPACE}
 # Set the value of $OVN_KUTTL_NAMESPACE if you want to run the ovn
 # kuttl tests in a namespace different than the default (ovn-kuttl-tests)
-ovn_kuttl: input deploy_cleanup ovn ovn_deploy_prep ## runs kuttl tests for the ovn operator. Installs ovn operator and cleans up previous deployments before running the tests, add cleanup after running the tests.
+ovn_kuttl: input deploy_cleanup ovn ovn_deploy_prep infra ## runs kuttl tests for the ovn operator. Installs ovn operator and cleans up previous deployments before running the tests, add cleanup after running the tests.
```

Comment:
instead can do as below so it also wait for infra operator to be ready before moving forward:-
	$(eval $(call vars,$@,infra))
	make wait
	
And also need do add $(eval $(call vars,$@,infra)) and  make infra_cleanup after ovn_cleanup

Reply:
Okay, done thanks